packages/@rnw-scripts/metro-dev-config/metro.dev.config.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ * @ts-check
+ */
+
+const {mergeConfig} = require('metro-config');
+
+const MetroConfig = require('@rnx-kit/metro-config');
+const {MetroSerializer} = require('@rnx-kit/metro-serializer');
+const {
+  DuplicateDependencies,
+} = require('@rnx-kit/metro-plugin-duplicates-checker');
+
+function makeMetroConfig(customConfig = {}) {
+  return mergeConfig(MetroConfig.makeMetroConfig(customConfig), {
+    resolver: {
+      blockList: MetroConfig.exclusionList([
+        // This prevents "react-native run-windows" from hitting: EBUSY: resource
+        // busy or locked for files produced by MSBuild
+        /.*\/build\/.*/,
+        /.*\/target\/.*/,
+        /.*\.tlog/,
```

Comment:
Should these be added to `@rnx-kit/metro-config`: https://github.com/microsoft/rnx-kit/blob/af6fb08c4678b3e74de9a59e10e9ec6f63bc7c00/packages/metro-config/src/index.js#L147-L159